core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -494,7 +506,13 @@ vaddr_t *unw_get_kernel_stack(void)
 	state.registers[FP] = read_fp();
 	state.registers[SP] = read_sp();
 	state.registers[LR] = read_lr();
-	state.registers[PC] = (uint32_t)unw_get_kernel_stack;
+
+	/*
+ 	 * Add 4 to make sure that we have an address well inside this function.
```

Comment:
Travis is complaining because there's a space character before the tab character.